docs/doc_start/30_myfirstproject.rst

```diff
@@ -0,0 +1,154 @@
+  Since we want to have the flexibility to work on different versions of our software, we use virtual environments.
+  You can find more information about pipenv `here <https://pipenv.pypa.io/en/latest/>`_.
+
+  .. code-block:: bash
+
+    pipenv install --dev
+
+5. Check your installation success.
+
+  .. code-block:: bash
+
+    nano Pipfile
+
+
+  Your output should look like this.
+
+  .. code-block:: bash
+
+    [[source]]
+    url = "https://pypi.org.sample"
+    verify_ssl = true
+    name = "pypi"
+
+    [packages]
+    viur-core = "==3.3.5"
+
+    [dev-packages]
+    viur-cli = "==0.7.7"
+    pycodestyle = "==2.10.0"
+    watchdog = "==2.2.1"
+
+    [requires]
+    python_version = "3.11"
+
+6. Start your pipenv
+
+  .. code-block:: bash
+
+    pipenv shell
+
+  This command will start a new virtual environment, installing packages in this environment will not install anything globally.
+  So feel free to fool around and experiment with packages.
+
+7. Finish you GCloud setup
+
+  .. code-block:: bash
+
+    #This script enables API's and configures some more appengine stuff
+    ./viur-gcloud-setup.sh projectname
+
+8. Try out your installation by running your dev server
+
+ .. code-block:: bash
+
+    viur run
+
+ .. warning::
+
+    | On the first start of your development server your console will show you an admin E-Mail and a generated admin password.
+    | You can log into the Vi and change the password by hand afterwards.
+    | It will look similar to this:
+
+ .. code-block:: bash
+
+    #ViUR created a new admin-user for you!
+    #Username: admin@projectname-viur.com
+    #Password: ThisIsARandomStr1ng!
+
+9. Open http://localhost:8080 and http://localhost:8080/vi to check if the front and backend of your system is working.
+
+10. After your first deployment open https://appspot.com
+11. Visit the `IAM & Admin <https://console.cloud.google.com/iam-admin/iam?>`_ and press the "+ GRANT ACCESS" Button
+12. Select the user: "projectname@appspot.gserviceaccount.com"
+13. Assign following roles:
+  (a) Cloud Datastore User
+  (b) Storage Object Admin
+
+Reset my Admin User:
+____________________
+In case you have locked yourself out of your own system. There is a possible way back in, but you will need to sacrifice
+all your user data and log ins.
+
+.. warning::
+    All your user data will be deleted in the Appengine and it will not be recoverable.
+    We only recommend this, if you see no other option of logging into your administration system
+
+.. Note::
+    The ViUR-Developers recommmend the usage of a Password Manager
+
+1. Open https://console.cloud.google.com and navigate to your project
+2. Navigate to the `Datastore Entities Site <https://console.cloud.google.com/datastore/databases/-default-/entities>`_ of your project.
+3. Search and delete all entries of the kind *user*
+4. Search and delete all entries of the kind *user_name_uniquePropertyIndex*
+5. Start your dev server again

```

Comment:
This is the beginner's way. It would be easier to simply add "root" to the admin user's (or another user's) `access` property of the apprpopriate Google Cloud Datastore entity. Generally, my advise is to entirely remove this section.

Reply:
I would like to keep it in. It is a safe, administrative way to reset your users and it is better to give people a solution to this specific problem instead of letting them try it alone.

Reply:
ok